index.html

```diff
@@ -1288,6 +1286,137 @@ <h2>DID Subject</h2>
+       does not resolve directly to a <a>representation</a> of the resource
+       identified by the <a>DID</a> (the <a>DID subject</a>). Rather the
+       <a>DID resolution</a> process produces a <a>DID document</a> which in
+       turn describes the <a>DID subject</a>.
+        </p>
+        <p>
+        Appendix A classifies the <a>resources</a> identified by <a>DIDs</a>
+        into three broad categories. In the first two, the <a>DID subject</a>
+        subject is not an information resource but an entity that exists
+        independent of a digital <a>representation</a> on a network. In the
+        third, the <a>DID subject</a>  is an information resource that has a
+        digital <a>representation</a>. Common examples of this type of resource
+        include web pages, PDF files, data schemas, or digital files of any
+        type.
+        </p>
+        <p>
+        In this third case, a <a>DID controller</a> MAY use the
+        <code>representation</code> property to: 1) include a <a>representation</a>
+        of the <a>DID subject</a> directly within the <a>DID document</a>,
+        and/or 2) reference a <a>representation</a> of the <a>DID subject</a>
+        directly from the <a>DID document</a>. Since an information resource
+        may have multiple <a>representations</a>, these two options are not
+        mutually exclusive.
+        </p>
+
+        <dl>
+        <dt>representation</dt>
+        <dd>
+        The <code>representation</code> property MAY have three optional
+        subproperties: <code>type</code>, <code>uri</code>, and <code>value</code>.
+        </dd>
+        </dl>
+
+        <dl>
+        <dt>type</dt>
+        <dd>
+        The <code>type</code> property MUST be a <a>URI</a> that asserts the
+        media type of the <a>representation</a>. The value of this property
+        SHOULD be a media type from the
+        <a href="https://www.iana.org/assignments/media-types/media-types.xhtml">IANA Media Type list </a>.
+        </dd>
+        </dl>
+
+        <dl>
+        <dt>uri</dt>
+        <dd>
+        The <code>uri</code> property MUST be a <a>URI</a>. This <a>URI</a>
+        SHOULD be a reference to a <a>representation</a> of the <a>DID subject</a>
+        that is not contained within the <a>DID document</a> but at some other
+        network location.
+        </dd>
+        </dl>
+
+        <dl>
+        <dt>value</dt>
+        <dd>
+        The <code>value</code> property contains a <a>representation</a> of
+        the <a>DID subject</a>. This value MUST first be encoded in the media
+        type indicated by the <code>type</code> subproperty. If necessary, the
+        value MUST then be re-encoded according to the requirements of the
+        <a>DID document</a> <a>representation</a> itself. See Section
+        <a href="#core-representations"></a>.
+        </dd>
+      </dl>
+
+        <p>
+        If both the <code>uri</code> property and the <code>value</code>
```

Comment:
Hrm, no, we should be using data URIs if we want to embed values directly... and we should put that value in `id` (see above). Also see https://en.wikipedia.org/wiki/Data_URI_scheme for explanation on data URIs.